app/modules/import-export/processes/import.php

```diff
@@ -461,16 +460,26 @@ private function get_default_settings_include() {
 		return [ 'templates', 'plugins', 'content', 'settings' ];
 	}
 
+	/**
+	 * Get the data that requires updating/replacement when imported.
+	 *
+	 * @return array
```

Comment:
Just for your knowledge, you don't have to do it, it might be helpful with IDE autocomplete.
```suggestion
	 * @return array{post_ids: array, term_ids: array}
```

Reply:
True.
In this case won't really help in anything, but good to start using it as a habit.